bin/cml/runner.js

```diff
@@ -333,23 +323,26 @@ const runLocal = async (opts) => {
 };
 
 const run = async (opts) => {
+  process.on('unhandledRejection', (reason) =>
```

Comment:
the event `uncaughtException` should also receive these events?

Reply:
No, uncaught is for not promisified errors while the other captures promises. Both are needed

Reply:
doesn't really matter but the docs say otherwise https://nodejs.org/dist/latest-v16.x/docs/api/process.html#event-uncaughtexception

Reply:
Umm Its true that it says that... That must be a new change but I can state that the one that captured a bug while writing the new state parser was the `unhandledRejection`